closure/compiler/closure_js_binary.bzl

```diff
@@ -101,6 +101,8 @@ def _impl(ctx):
              "--define=goog.json.USE_NATIVE_JSON"]
   if ctx.attr.output_wrapper:
     args += ["--output_wrapper=%s" % ctx.attr.output_wrapper]
+    if ctx.attr.output_wrapper == "(function(){%output%}).call(this);":
```

Comment:
It isn't clear to me how many variants of output_wrappers we should try to support (eg: extra whitespace, etc). Also, if compiler level module support is later added we would need to support more complex module_wrapper patterns (eg: `var __sg__=__sg__||{};(function(a){%s}).call(this, __sg__);`). Currently the only compelling use case that I know of for output_wrappers are function wrapper. Do you know of any others?

If you don't know of others, does special casing this as the canonical output_wrapper seem like reasonable behavior to you?

FYI, if this were to become the default output_wrapper then the special case seems less surprising.